resources/language/resource.language.fr_fr/strings.po

```diff
@@ -174,75 +174,75 @@ msgstr "Activer les vues personnalisées"
 
 msgctxt "#30039"
 msgid "View for folders"
-msgstr "Voir les dossiers"
+msgstr "Vue pour les dossiers"
 
 msgctxt "#30040"
 msgid "View for movies"
-msgstr "Voir les films"
+msgstr "Vue pour les films"
 
 msgctxt "#30041"
 msgid "View for shows"
-msgstr "Voir les séries"
+msgstr "Vue pour les séries"
 
 msgctxt "#30042"
 msgid "View for seasons"
 msgstr "Vue pour les saisons"
 
 msgctxt "#30043"
 msgid "View for episodes"
-msgstr "Voir les épisodes"
+msgstr "Vue pour les épisodes"
 
 msgctxt "#30044"
 msgid "View for profiles"
-msgstr "Voir les profils"
+msgstr "Vue pour les profils"
 
 msgctxt "#30045"
 msgid "Thumb rating removed|You rated a thumb down|You rated a thumb up"
-msgstr "Évaluation supprimée|Évaluation négative|Évaluation positive"
+msgstr "Evaluation supprimée|'Pouce baissé' enregistré|'Pouce levé' enregistré"
```

Comment:
Question/Suggestion: seem not reflect original purpose, an alternative?

Reply:
Actually the initial reason why I updated this line is because for me the idea of "action" was missing in the sentence but simply adding `enregistré` to the original translation would have been too long according to me (i.e. `Évaluation négative enregistrée`). Then I also decided to stick more to the original meaning because, in case you didn't translate it already, `pouce baissé` means `thumb down` and `pouce levé` means `thumb up` literally.

Why do you think it does not reflect the original purpose?

Reply:
the message is shown like a popup the lenght is not a limit
others translations have varied the sentence by using other words like
"you voted like it" etc ..

I just wanted to clarify because often happened that new translators invalidate sentences translated by previous translators, and then discussions arise between translators

Reply:
I understand your question.
Since Netflix does not provide a word for the ranking on their website (only the thumb down/up icon is displayed) I would prefer to keep the litteral translation of thumb down/up.
And I am open to discussion with previous translators as I hope they will understand I updated the translation to be more precise, not to change their work 😃

Your comment on this line made me notice I removed an accent on the first letter by mistake. I will correct it in my next commit.